product_docs/docs/pgd/5/durability/limitations.mdx

```diff
@@ -43,10 +43,10 @@ nodes in a group. If you use this feature, take the following limitations into a
 
 ## Eager
 
-[Eager](../consistency/eager) is available through Group Commit. It avoids conflicts by eagerly aborting transactions that might clash. It's subject to the same limitations as Group Commit.
+[Eager](../consistency/eager) is available through Group Commit. It avoids conflicts by eagerly aborting transactions that might clash. It's subject to the same limitations as Group Commit. In addition:
```

Comment:
There was nothing introducing this single bullet. Is this what was meant? Or can we just take out the bullet  (making it a regular paragraph) and the words "in addition" that I added?

Reply:
@djw-m I think Betsy's edit here makes sense. 
Alternatively, we could make a Note or Important note taking part of the text and the bullet:
```
Important 
Eager is subject to the same limitations as Group Commit. Additionally, it doesn't allow the `NOTIFY` SQL command or the `pg_notify()` function. It also doesn't allow `LISTEN` or `UNLISTEN`.
```

Reply:
Take out the bullet. I think that was a side effect of these being extracted from elsewhere. (And lose the In Addition)